### ISSUE TITLE
"Uncaught TypeError: Cannot read property 'tables' of undefined" when clicking in embedded chart

### ISSUE THREAD

**Issue by c0bra** [metabase/metabase#7641]
I get this error in the console any time I click in a chart in an embedded dashboard view. The tooltip pops up initially but fades and then stays on the screen. It does not disappear and new tooltips do not appear when I click on other elements, but this error continues to show up in the console on each click.

This seems to be the same issue as this one: http://discourse.metabase.com/t/when-embedded-hover-tip-locks-viz-on-select-js-error-after/2425

```
StructuredQuery.js:223 Uncaught TypeError: Cannot read property 'tables' of undefined
    at StructuredQuery.table (StructuredQuery.js:223)
    at Question.tableMetadata (Question.js:339)
    at Function.forQuestion (Mode.js:28)
    at Question.mode (Question.js:345)
    at Visualization.getClickActions (Visualization.jsx:269)
    at Visualization.render (Visualization.jsx:319)
    at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (ReactCompositeComponent.js:796)
    at ReactCompositeComponentWrapper._renderValidatedComponent (ReactCompositeComponent.js:819)
    at ReactCompositeComponentWrapper._updateRenderedComponent (ReactCompositeComponent.js:743)
    at ReactCompositeComponentWrapper._performComponentUpdate (ReactCompositeComponent.js:721)]
```

**Browser:** Chrome 66.0.3359.139
**OS:** macOS Sierra
**Metabase version:**  0.28.6 
**Metabase hosting environment**: docker
**Metabase internal database:** H2

**Comment by paubelsan** [comment]
+1

**Comment by omkardusane** [comment]
+1

**Comment by pralok** [comment]
+1

**Comment by salsakran** [comment]
Hello all, this is fixed in 0.32.10 which just went out.
